.github/workflows/nightly-build.yml

```diff
@@ -84,41 +84,30 @@ jobs:
         include:
         - target: aarch64-apple-darwin
           os: macos-latest
-          target_rustflags: ''
```

Comment:
I'm pretty sure that not having `target_rustflags: ''` causes winget to fail. there's a closed issue on it in the repo somewhere. don't ask me to explain it.